### PR TITLE
Useful scripts

### PR DESCRIPTION
**Pull request by ajdapretnar**
The first batch of scripts available for use in Python Script widget.
Each script has a short header, which states the input and the output of the script and which add-ons it requires.
Scripts are listed in README.

### CODE REVIEW

File: scripts/Text/sentences-to-corpus.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+tokens = in_data.tokens
+new_domain = Domain(attributes=[], metas=[StringVariable('Sentences'),
+                                          StringVariable('Title')])
+
+titles = []
+content = []
+
+for i, doc in enumerate(tokens):
+    for t in doc:
+        titles.append(in_data[i]['Title'].value)
```
Comment (PrimozGodec) [inline scripts/Text/sentences-to-corpus.py]: If an attribute with the name `Title` is missing this script will fail. Maybe you can take a column which has a `title` attribute set to True and use it as a title here.

Comment (ajdapretnar) [reply scripts/Text/sentences-to-corpus.py]: I am not sure my fix is ok. Please check.

File: scripts/Text/sentences-to-corpus.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+requires: Text add-on
+"""
+
+import numpy as np
+from Orange.data import Domain, StringVariable
+from orangecontrib.text.corpus import Corpus
+
+tokens = in_data.tokens
+new_domain = Domain(attributes=[], metas=[StringVariable('Sentences'),
+                                          StringVariable('Title')])
```
Comment (PrimozGodec) [inline scripts/Text/sentences-to-corpus.py]: Maybe just here use the same title variable than it is in original data. Just replace this line with `title])` and move title definition from line 17 to line above this one.

File: scripts/Text/sentences-to-corpus.py (hunk 1 above)
Comment (PrimozGodec) [reply scripts/Text/sentences-to-corpus.py]: It is great. 👍  I just suggested one small correction, to persist the same title variable.

File: scripts/Text/sentences-to-corpus.py (hunk 2 above)
Comment (ajdapretnar) [reply scripts/Text/sentences-to-corpus.py]: I was thinking about that, but wasn't sure if it is ok to reuse variables.

**Pull request merged**